tests/integration_tests/README.md

```diff
@@ -1,52 +1,101 @@
+### Run integration tests locally
 
-    * `bin/tidb-server` # version >= 4.0.0-rc.1
-    * `bin/tikv-server` # version >= 4.0.0-rc.1
-    * `bin/pd-server`   # version >= 4.0.0-rc.1
-    * `bin/pd-ctl`      # version >= 4.0.0-rc.1
-    * `bin/tiflash`     # needs tiflash binary and some necessary so files
-    * `bin/sync_diff_inspector`
-    * [bin/go-ycsb](https://github.com/pingcap/go-ycsb)
-    * [bin/etcdctl](https://github.com/etcd-io/etcd/tree/master/etcdctl)
-    * [bin/jq](https://stedolan.github.io/jq/)
-    * [bin/minio](https://github.com/minio/minio)
+1. The following executables must be copied or generated or linked into these locations, `sync_diff_inspector` can be
+   downloaded
+   from [tidb-community-toolkit](https://download.pingcap.org/tidb-community-toolkit-v4.0.2-linux-amd64.tar.gz)
+   , `tidb-server` related binaries can be downloaded
+   from [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz):
 
-    > If you are running tests on MacOS, tidb related binaries can be downloaded from tiup mirrors, such as https://tiup-mirrors.pingcap.com/tidb-v4.0.2-darwin-amd64.tar.gz. And `sync_diff_inspector` can be compiled by yourself from source [tidb-tools](https://github.com/pingcap/tidb-tools)
+   * `bin/tidb-server` # version >= 4.0.0-rc.1
+   * `bin/tikv-server` # version >= 4.0.0-rc.1
+   * `bin/pd-server`   # version >= 4.0.0-rc.1
+   * `bin/pd-ctl`      # version >= 4.0.0-rc.1
+   * `bin/tiflash`     # needs tiflash binary and some necessary so files
+   * `bin/sync_diff_inspector`
+   * [bin/go-ycsb](https://github.com/pingcap/go-ycsb)
+   * [bin/etcdctl](https://github.com/etcd-io/etcd/tree/master/etcdctl)
+   * [bin/jq](https://stedolan.github.io/jq/)
+   * [bin/minio](https://github.com/minio/minio)
 
-    > All Tiflash required files can be found in [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz) packages. You should put `flash_cluster_manager`, `libtiflash_proxy.so` and `tiflash` into `bin` directory in TiCDC code base.
+   > If you are running tests on MacOS, tidb related binaries can be downloaded from tiup mirrors, such as https://tiup-mirrors.pingcap.com/tidb-v4.0.2-darwin-amd64.tar.gz. And `sync_diff_inspector` can be compiled by yourself from source [tidb-tools](https://github.com/pingcap/tidb-tools)
+
+   > All Tiflash required files can be found in [tidb-community-server](https://download.pingcap.org/tidb-community-server-v4.0.2-linux-amd64.tar.gz) packages. You should put `flash_cluster_manager`, `libtiflash_proxy.so` and `tiflash` into `bin` directory in TiCDC code base.
 
 2. The following programs must be installed:
 
-    * `mysql` (the MySQL cli client, currently [mysql client 8.0 is not supported](https://github.com/pingcap/tidb/issues/14021))
+   * `mysql` (the MySQL cli client,
+     currently [mysql client 8.0 is not supported](https://github.com/pingcap/tidb/issues/14021))
```

Comment:
Could u add some normal way to install mysql client by yum/apt/brew? So I can install mysql client by one step(copy and paste).
like: 
```
sudo yum install mysql-community-client.x86_64 -y
```

Reply:
I found it difficult to explain these commands in our documentation, so I used the official documentation link.